src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -135,8 +135,6 @@ protected function setMappingDriverConfig(array $mappingConfig, string $mappingN
      *
      * Returns false when autodetection failed, an array of the completed information otherwise.
      *
-     * @param string|null $bundleDir The bundle directory path
```

Comment:
FYI, these are false-positive for us: we use `@param` to tell about parameters we'll add in the next major (see also the special comment in the signature)